extension/main.js

```diff
@@ -13,6 +13,15 @@
 		chrome.browserAction.setTitle({
 			title: title
 		});
+
+		if (badge && window.GitHubNotify.settings.get('showDesktopNotif')) {
+			chrome.notifications.create('github-notifier', {
+				title: 'GitHub',
+				iconUrl: 'icon-128.png',
+				type: 'basic',
+				message: 'You have new unread notifications'
```

Comment:
We could then use the `buttons` to have a button for `View` and one for `Mark as read`.